dojo/tools/twistlock/parser.py

```diff
@@ -1,16 +1,118 @@
+import io
+import csv
 import json
+import hashlib
+
 from dojo.models import Finding
 
 
-class TwistlockParser(object):
-    def __init__(self, json_output, test):
 
-        tree = self.parse_json(json_output)
 
+class TwistlockCSVParser(object):
+
+    @staticmethod
+    def get_field_from_row(row, column):
+        if row[column] is not None:
+            return row[column]
+        else:
+            return None
+
+    def parse_issue(self, row, test):
+
+        if not row:
+            return None
+
+        id_column = 4
+        line_column = 5
+        code_line_column = 6
+        vulnerability_id_column = 10
+        severity_column = 13
+        package_name_column = 14
+        package_version_column = 16
+        cvss_column = 18
+        fix_status_column = 19
+        description_column = 20
+
+        if self.get_field_from_row(row, vulnerability_id_column) is None:
```

Comment:
I think these ifs statements would be more readable just writing:
```
 data_vulnerability_id  = self.get_field_from_row(row, vulnerability_id_column
 if data_vulnerability_id is None:
   data_vulnerability_id = '' 
```

Reply:
or maybe you can write an helper method:

```
def get_field_from_row_or_default(row, column, default):
   field := self.get_field_from_row(row, column)
    if filed is None:
      return default
    return field
```
and then you can invoke this helper for all fields with a default value:
```
data_vulnerability_id = self.get_field_from_row_or_default(row, vlunerability_id_column, '')
```

Reply:
I did some refactor, would you check it?